### PR TITLE
Removed unused app component css file

### PR DESCRIPTION
**Pull request by westerncj**
Summary of the changes (Less than 80 chars)
 - Removed unused css for app component

### CODE REVIEW

File: src/Templating/src/Microsoft.DotNet.Web.Spa.ProjectTemplates/content/Angular-CSharp/ClientApp/src/app/app.component.css [hunk 1]
```diff
@@ -1,6 +1 @@
-@media (max-width: 767px) {
-  /* On small screens, the nav menu spans the full width of the screen. Leave a space for it. */
-  .body-content {
-    padding-top: 50px;
-  }
-}
+
```
Comment (natemcmaster) [inline src/Templating/src/Microsoft.DotNet.Web.Spa.ProjectTemplates/content/Angular-CSharp/ClientApp/src/app/app.component.css]: I don't know how Angular works, but can we delete the empty file altogether?

Comment (damienbod) [reply src/Templating/src/Microsoft.DotNet.Web.Spa.ProjectTemplates/content/Angular-CSharp/ClientApp/src/app/app.component.css]: You need to remove it from the ts file then:

https://github.com/aspnet/AspNetCore/blob/master/src/Templating/src/Microsoft.DotNet.Web.Spa.ProjectTemplates/content/Angular-CSharp/ClientApp/src/app/app.component.ts#L6

Comment (westerncj) [reply src/Templating/src/Microsoft.DotNet.Web.Spa.ProjectTemplates/content/Angular-CSharp/ClientApp/src/app/app.component.css]: I can take care of this. Thanks!

**Comment by natemcmaster** [comment]
@ryanbrandenburg are you okay with this?

**Comment by ryanbrandenburg** [comment]
That CSS is used and affects the result as far as I can tell. @westerncj could you go into more detail about why you think it's unused?

**Comment by westerncj** [comment]
@ryanbrandenburg, I originally deleted the `.body-content` class from the `app.component.css` file since the css class wasn't used anywhere. After removing it, @natemcmaster thought maybe we should go ahead delete the empty file.

**Review by ryanbrandenburg — APPROVED** [review]

**Comment by ryanbrandenburg** [comment]
@westerncj you're correct, body-content isn't used anywhere, I misread it as body when investigating. If you could rebase on to latest I'll be able to merge as soon as the tests pass.

Thanks!

**Comment by westerncj** [comment]
Thanks @ryanbrandenburg. I rebased to the latest.

**Pull request merged**